src/Files.App/Actions/Navigation/PreviousTabAction.cs

```diff
@@ -0,0 +1,44 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.DependencyInjection;
+using Files.App.Commands;
+using Files.App.Contexts;
+using Files.App.Extensions;
+using System.ComponentModel;
+using System.Threading.Tasks;
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class PreviousTabAction : ObservableObject, IAction
+	{
+		private readonly IMultitaskingContext multitaskingContext = Ioc.Default.GetRequiredService<IMultitaskingContext>();
+
+		public string Label { get; } = "PreviousTab".GetLocalizedResource();
+
+		public string Description { get; } = "TODO: Need to be described.";
+
+		public bool IsExecutable => multitaskingContext.TabCount > 1;
+
+		public HotKey HotKey { get; } = new(VirtualKey.Tab, VirtualKeyModifiers.Control | VirtualKeyModifiers.Shift);
+
+		public PreviousTabAction()
+		{
+			multitaskingContext.PropertyChanged += MultitaskingContext_PropertyChanged;
+		}
+
+		public Task ExecuteAsync()
+		{
+			if (App.AppModel.TabStripSelectedIndex is 0)
+				App.AppModel.TabStripSelectedIndex = multitaskingContext.TabCount - 1;
+			else
+				App.AppModel.TabStripSelectedIndex--;
+			return Task.CompletedTask;
```

Comment:
```suggestion

			return Task.CompletedTask;
```

Add a new line break before return statement